store/src/java/com/zimbra/cs/datasource/imap/ImapFolderSync.java

```diff
@@ -350,6 +356,13 @@ public void syncMessages() throws ServiceException, IOException {
             }
         }
 
+        // ZBUG-2361: this is to push local draft changes into remote 
```

Comment:
You don't need to list the ticket here.
```suggestion
        // Push local draft changes into remote 
```

Reply:
done